library/src/main/java/com/novoda/downloadmanager/lib/BatchStatusRepository.java

```diff
@@ -78,4 +78,31 @@ int getBatchStatus(long batchId) {
         return STATUS_UNKNOWN_ERROR;
 
     }
+
+    long getBatchSizeInBytes(long batchId) {
+        Cursor cursor = null;
+        long totalSize = 0;
+        try {
+            String[] selectionArgs = {String.valueOf(batchId)};
+            cursor = resolver.query(ALL_DOWNLOADS_CONTENT_URI,
```

Comment:
We only have one column to sum so it can be in the projection? Then it's not much of a raw query